canonical-web-and-design/conjure-up.io
# Move from Jekyll to 11ty

## Done

- Move Jekyll to 11ty

## QA

Changes should not affect the site, therefore everything should be as usual.

- Checkout this branch
- `dotrun clean && dotrun`
- Check that everything looks the same as https://conjure-up.io/
- Make sure JS libraries are not missing


To make sure we don't have any issues on prod test dockerfile too

- Build docker image: `DOCKER_BUILDKIT=1 docker build --tag conjure-up .`
- Run docker image: `docker run -ti -p 8111:80 conjure-up`
- Visit localhost:8111
- Make sure JS libraries are not missing

## Issue / Card

Fixes #75
